### PR TITLE
Tour printing enhancements and 22.8.0 french translation

### PR DESCRIPTION
**Pull request by FJBDev**
- Added the ability to enable/disable the opening of the printed tour in the default PDF viewer
- Improved french mnemonics in the Tour Printing Dialog
- **22.8** French translation

### CODE REVIEW

File: bundles/net.tourbook/src/net/tourbook/messages.properties [hunk 1]
```diff
@@ -615,7 +616,7 @@ Dialog_GotoMapLocation_Label_LocationLatitude                          = L&atitu
 Dialog_GotoMapLocation_Label_LocationLongitude                         = L&ongitude
 Dialog_GotoMapLocation_Message_LatLonIsPasted_FromClipboard            = Latitude and longitude was filled from the clipboard:\n\
                                                                          {0}
-Dialog_GotoMapLocation_Message_LatLonIsPasted_FromMouseMapPosition     = Latitude and longitude was filled from the current mouse map position 
+Dialog_GotoMapLocation_Message_LatLonIsPasted_FromMouseMapPosition     = Latitude and longitude were filled from the current mouse map position
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/messages.properties:619]: My first version was "Latitude and longitude is filled ..."

I'm always struggeling with some words :-)

File: bundles/net.tourbook.printing/src/net/tourbook/printing/PrintTourPDF.java [hunk 2]
```diff
@@ -126,129 +131,109 @@ public void printPDF(final IXmlSerializable object, final PrintSettings printSet
...
+      transformer.setParameter("lang.Tour_Print_Maximum_Pulse", Messages.Tour_Print_Maximum_Pulse); //$NON-NLS-1$
+      transformer.setParameter("lang.Tour_Print_Average_Pulse", Messages.Tour_Print_Average_Pulse); //$NON-NLS-1$
+      transformer.setParameter("lang.Tour_Print_Calories", Messages.Tour_Print_Calories); //$NON-NLS-1$
+      transformer.setParameter("lang.Tour_Print_Average_Cadence", Messages.Tour_Print_Average_Cadence); //$NON-NLS-1$
+      transformer.setParameter("lang.Tour_Print_Altitude", Messages.Tour_Print_Altitude); //$NON-NLS-1$
+      transformer.setParameter("lang.Tour_Print_Highest_Altitude", Messages.Tour_Print_Highest_Altitude); //$NON-NLS-1$
+      transformer.setParameter("lang.Tour_Print_Meters_Up", Messages.Tour_Print_Meters_Up); //$NON-NLS-1$
+      transformer.setParameter("lang.Tour_Print_Meters_Down", Messages.Tour_Print_Meters_Down); //$NON-NLS-1$
+      transformer.setParameter("lang.Tour_Print_Tour_Markers", Messages.Tour_Print_Tour_Markers); //$NON-NLS-1$
+      transformer.setParameter("lang.Tour_Print_No_Markers_Found", Messages.Tour_Print_No_Markers_Found); //$NON-NLS-1$
```
Comment (wolfgang-ch) [inline bundles/net.tourbook.printing/src/net/tourbook/printing/PrintTourPDF.java:237]: When I would work on this file and would see this pile of compacted characters, I would reformat it manually into columns as this is done automatically for fields to make it better readable

Comment (FJBDev) [reply bundles/net.tourbook.printing/src/net/tourbook/printing/PrintTourPDF.java:237]: Do you want me to do another PR to address that ?

File: bundles/net.tourbook/src/net/tourbook/messages.properties (hunk 1 above)
Comment (FJBDev) [reply bundles/net.tourbook/src/net/tourbook/messages.properties:619]: That's why it's good to have several "pair of eyes" looking at the code/translation 👍

File: bundles/net.tourbook.printing/src/net/tourbook/printing/PrintTourPDF.java (hunk 2 above)
Comment (wolfgang-ch) [reply bundles/net.tourbook.printing/src/net/tourbook/printing/PrintTourPDF.java:237]: > Do you want me to do another PR to address that ?

No

**Pull request merged**